azblob/list.go

```diff
@@ -94,6 +106,11 @@ func (azp *Storer) List(ctx context.Context, opts ...Option) (*ListerResponse, e
 		r.Prefix = *resp.Prefix
 	}
 
+	r.Marker = resp.NextMarker
```

Comment:
I wonder if we can use azurite to make a useful intergration test here to ensure paging works?

Putting as minor as this fix works for now, it can be a follow up PR.

Reply:
This is a good idea and we should do this at some point, but not in this PR please